decidim-participatory_processes/app/commands/decidim/participatory_processes/create_space_page.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    class CreateSpacePage < Decidim::Command
```

Comment:
I think the name of the class should refer to the feature, as it doesn't answer the question "which page should it create?" 

Maybe something like `CreateDemocraticQualityIndicatorsPage` ?